packages/elements-core/src/__fixtures__/services/with-url-variables.ts

```diff
@@ -47,37 +47,39 @@ export const httpServiceWithUrlVariables: IHttpService = {
       description: 'Development API',
     },
   ],
-  security: [],
-  securitySchemes: [
-    {
-      id: '?http-security?',
-      key: 'oauth_scheme',
-      type: 'oauth2',
-      description:
-        'Get access to data while protecting your account credentials. OAuth2 is also a safer and more secure way to give you access.',
-      flows: {
-        implicit: {
-          authorizationUrl: 'https://implicit.authorization-url.com',
-          refreshUrl: 'https://implicit.refresh-url.com',
-          scopes: { 'scope:implicit': 'implicit scope description' },
-        },
-        password: {
-          tokenUrl: 'https://password.token-url.com',
-          refreshUrl: 'https://password.refresh-url.com',
-          scopes: { 'scope:password': 'password scope description' },
-        },
-        clientCredentials: {
-          tokenUrl: 'https://clientCredentials.token-url.com',
-          refreshUrl: 'https://clientCredentials.refresh-url.com',
-          scopes: { 'scope:clientCredentials': 'clientCredentials scope description' },
-        },
-        authorizationCode: {
-          authorizationUrl: 'https://authorizationCode.authorization-url.com',
-          tokenUrl: 'https://authorizationCode.token-url.com',
-          refreshUrl: 'https://authorizationCode.refresh-url.com',
-          scopes: { 'scope:authorizationCode': 'authorizationCode scope description' },
+  securitySchemes: [],
+  security: [
```

Comment:
I'd keep it in `securitySchemes` as it was.
What we have right now is not quite valid - `flows` would never get set if `securitySchemes` didn't have that `oauth_scheme` defined.